build.gradle

```diff
@@ -48,6 +49,7 @@ allprojects {
   apply plugin: 'idea'
 
   repositories {
+    mavenLocal()
```

Comment:
Please don't commit this -- it is only for local test.

Reply:
Done.